### DISCUSSION TITLE
Tell PDM to use private index only for selected packages

### DISCUSSION THREAD

**Discussion by baggiponte** [pdm-project/pdm#1693] [Q&A] [answered]
Hi! This is a minor thing, I guess. Where I work, we are using a private index to store private packages, and the private index is the primary one (the other is pypi). This means that when I `pdm add/install/update`, every package is looked for in the private index first, then in pypi if it's not there. Since we plan to have just a handful of packages in the private index, I was wondering if there was a way to tell the package manager to look at the private index only for specific packages. Thanks!

**Comment by frostming** [comment, marked as answer]
May be duplicated with #1645
